apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs

```diff
@@ -68,13 +68,15 @@ protected override void Configure(IObjectTypeDescriptor<CalculationDto> descript
             .Resolve(context => context.Parent<CalculationDto>().OrchestrationState switch
             {
                 CalculationOrchestrationState.Scheduled => ProcessStatus.Neutral,
+                CalculationOrchestrationState.Started => ProcessStatus.Info,
                 CalculationOrchestrationState.Calculating => ProcessStatus.Info,
                 CalculationOrchestrationState.CalculationFailed => ProcessStatus.Danger,
                 CalculationOrchestrationState.Calculated => ProcessStatus.Info,
                 CalculationOrchestrationState.ActorMessagesEnqueuing => ProcessStatus.Info,
                 CalculationOrchestrationState.ActorMessagesEnqueuingFailed => ProcessStatus.Danger,
                 CalculationOrchestrationState.ActorMessagesEnqueued => ProcessStatus.Info,
                 CalculationOrchestrationState.Completed => ProcessStatus.Success,
+                CalculationOrchestrationState.Canceled => ProcessStatus.Warning,
```

Comment:
@ManBearTM I'm not sure if the `warning` color is correct to use here

Reply:
Don't know, but I went with neutral instead 😄